query-engine/connector-test-kit-rs/query-engine-tests/tests/new/occ.rs

```diff
@@ -112,7 +112,7 @@ mod occ {
         assert_eq!(booked_user_id, found_booked_user_id);
     }
 
-    #[connector_test(schema(occ_simple), exclude(MongoDB, CockroachDb))]
+    #[connector_test(schema(occ_simple), exclude(MongoDB, CockroachDb, TiDB))]
```

Comment:
Why does TiDBneed to be excluded from the OCC tests here? Issue?

Reply:
@Mini256  ?

Reply:
This should not be a TiDB problem, if I use mysql as the connector, occ's test passes, maybe there is some configuration I overlooked. I need to check it.

```json
{
    "connector": "mysql",
    "runner": "direct",
    "version": "5.7"
}
```

Reply:
😢 This is so strange. When I setting connector's `relationMode` to `ForeignKeys`, all the tests in OCC can pass.

Reply:
`relationMode=prisma` actually triggers additional logic on our side, so maybe for TiDB that just means that we have a case there that we need to look into. Does not really surprise me - I think we can leave it like this and later investigate what is going on.

Reply:
**Action** : @janpio  to investigate later with prisma team 

@janpio  please advice if that's the right understanding :)

Reply:
How exactly are these tests in this file failing @Mini256?

Reply:
Since the new version of TiDB already supports foreign key constraints, we can now use `relationMode=ForeignKeys` and all test cases related to OCC have passed.